### ISSUE TITLE
Support multiple deployments (contexts) via Coder CLI

### ISSUE THREAD

**Issue by francisco-mata** [coder/coder#9857]
Hello Coders,

I would like to request to have the CLI feature to be able to login on multiple coders at the same time. Currently to SSH to a coder WS you will need to login to server, to create a token, and with that token it will grant you access to your WS SSh connection. But If you need to connect to other Coder WS, you will need to relog to new instance of coder, killing the login token of the first instance. 

I had managed this error before by mapping on the config file {url1: token1, url2: token2}, and exposing a default server on CLI and a flag to override the location `-l server1` if needed. 

```
coder login server1
coder login server2
coder location set-default server1
coder ssh-config 
coder ssh-config -l server2
coder ssh coder-ws-1 
coder ssh coder-ws-2 -l server2
```

Let me know what you think about this approach. 

PS: We run multi-arch K8s clusters, so depends on which cluster you run, you will have different capabilities

**Comment by mattlqx** [comment]
Agree, we have a staging and prod deployment of Coder that our team maintains and it'd be great to have the ability to switch between the two.

**Comment by jpmh1309** [comment]
Any ETA for this feature!? :eyes:

**Comment by kylecarbs** [comment]
@jpmh1309 not yet, but we've tinkered with it. It's extremely easy for us to implement, UX is just a bit odd.

**Comment by bpmct** [comment]
closing in favor of https://github.com/coder/coder/issues/13325

**Comment by bpmct** [comment]
Some good discussion in https://github.com/coder/coder/issues/13325

**Comment by bjornrobertsson** [comment]
Adding comment from customer:
```
Because I use pure ssh to log into my workspaces I would like to have a system where I do not have to use a command to switch context. i like to have it by domain like ssh <ws>.<env>.coder.
```
